esx_service/vmdk_ops.py

```diff
@@ -696,20 +696,24 @@ def get_vol_path(datastore, tenant_name=None):
     # is created on <datastore>/DOCK_VOLS_DIR/tenant_uuid
     # a symlink <datastore>/DOCK_VOLS_DIR/tenant_name will be created to point to
     # path <datastore>/DOCK_VOLS_DIR/tenant_uuid
+    # If the command is under a tenant then the path returned contains tenant
+    # name not UUID. This is to make logs more readable. OS will resolve this path
+    # as a symlink with tenant_name is already present.
 
-    path = dock_vol_path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)
+    rpath = path = dock_vol_path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)
 
     if tenant_name:
         error_info, tenant = auth_api.get_tenant_from_db(tenant_name)
         if error_info:
             logging.error("get_vol_path: failed to find tenant info for tenant %s", tenant_name)
             path = dock_vol_path
         path = os.path.join(dock_vol_path, tenant.id)
+        rpath = os.path.join(dock_vol_path, tenant_name)
```

Comment:
why rpath? (in the sense, what's "r" stands for?

And it looks like rpath is in fact the same thing as the symlink_path on Line 739. Can you just use symlink_path here?